src/services/membership.ts

```diff
@@ -48,115 +57,76 @@ export type MembershipUpload = {
   Privacy: boolean;
 };
 
+const get = async (queryParams: {
+  involvementCode?: string;
+  username?: string;
+  sessionCode?: string;
+  participationTypes?: Participation[] | Participation;
+}): Promise<MembershipView[]> => http.get(`memberships/${http.toQueryString(queryParams)}`);
```

Comment:
You use a `/` before the query string here whereas in emails.ts you do not (line 18). Is there a reason for this inconsistency or can we keep it the same in one pull request?

Reply:
Ah, it shouldn't have the slash, that's leftover from a previous url that used route params instead of query params. Thanks for catching that!